cli/mngcheckpoint/mngcheckpoint.go

```diff
@@ -0,0 +1,136 @@
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with spaces: %s", err.Error())
+	}
+	if string(spacesJson) != "null" {
+		os.Setenv("TT_CLI_CAT_SPACES", string(spacesJson))
+	}
+
+	os.Setenv("TT_CLI_CAT_FROM", strconv.FormatUint(catFlag.From, 10))
+	os.Setenv("TT_CLI_CAT_TO", strconv.FormatUint(catFlag.To, 10))
+
+	// List of replicas is passed to lua cat scipt via environment variable in json format.
+	replicasJson, err := json.Marshal(catFlag.Replica)
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with replicas: %s", err.Error())
+	}
+	if string(replicasJson) != "null" {
+		os.Setenv("TT_CLI_CAT_REPLICAS", string(replicasJson))
+	}
+
+	return nil
+}
+
+// FillPlayEnv fills the internal env vars of play cmd with cli args.
+func FillPlayEnv(args []string, playFlag CmdFlags) error {
+	if len(args) < 2 {
+		return fmt.Errorf("It is required to specify an URI and at least one .xlog or .snap file.")
+	}
+
+	// List of files and URI is passed to lua play scipt via environment variable in json format.
+	filesAndUriJson, err := json.Marshal(args)
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with files and uri: %s", err.Error())
+	}
+
+	os.Setenv("TT_CLI_PLAY_FILES_AND_URI", string(filesAndUriJson))
+	os.Setenv("TT_CLI_PLAY_SHOW_SYS", strconv.FormatBool(playFlag.ShowSystem))
+
+	// List of spaces is passed to lua play scipt via environment variable in json format.
+	spacesJson, err := json.Marshal(playFlag.Space)
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with spaces: %s", err.Error())
+	}
+	if string(spacesJson) != "null" {
+		os.Setenv("TT_CLI_PLAY_SPACES", string(spacesJson))
+	}
+
+	os.Setenv("TT_CLI_PLAY_FROM", strconv.FormatUint(playFlag.From, 10))
+	os.Setenv("TT_CLI_PLAY_TO", strconv.FormatUint(playFlag.To, 10))
+
+	// List of replicas is passed to lua play scipt via environment variable in json format.
+	replicasJson, err := json.Marshal(playFlag.Replica)
+	if err != nil {
+		log.Fatalf("Internal error: problem with creating json params with replicas: %s", err.Error())
+	}
+	if string(replicasJson) != "null" {
+		os.Setenv("TT_CLI_PLAY_REPLICAS", string(replicasJson))
+	}
+
+	return nil
+}
+
+// Cat returns the contents of .snap/.xlog files.
+func Cat(cmdCtx *cmdcontext.CmdCtx) (stdout []byte, err error) {
+	var outbuff, errbuff bytes.Buffer
+	cmd := exec.Command(cmdCtx.Cli.TarantoolExecutable, "-e", catFile)
+	cmd.Stdout, cmd.Stderr = &outbuff, &errbuff
+	if err := cmd.Run(); err != nil {
+		return outbuff.Bytes(), fmt.Errorf("Result of cat: %s", errbuff.String())
+	}
+
+	return outbuff.Bytes(), nil
+}
+
+// Play returns the result of playing the contents of .snap/.xlog files to another Tarantool instance.
+func Play(cmdCtx *cmdcontext.CmdCtx) (stdout []byte, err error) {
+	var outbuff, errbuff bytes.Buffer
+	cmd := exec.Command(cmdCtx.Cli.TarantoolExecutable, "-e", playFile)
```

Comment:
Now we prefer to use `stdin` instead of `-e`. (https://github.com/tarantool/tt/commit/d32eadb11338800475591e6a5234128974e79023)

Reply:
Done